docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,282 @@
+---
+jupytext:
+ text_representation:
+   extension: .md
+   format_name: myst
+   format_version: 0.13
+   jupytext_version: 1.11.1
+kernelspec:
+ display_name: Python 3
+ language: python
+ name: python3
+---
+
+
+# What additional options are available when using LRE?
+
+
+In [](lre-1-intro.md), {func}`.execute_with_lre` was used to calculated the error mitigated expectation values
+In this section, we will outline the optional arguments that can be used and adjusted with this technique.
+
+
+```python
+lre_value = mitiq.lre.execute_with_lre(
+    circuit,
+    executor,
+    degree,
+    fold_multiplier,
+    folding_method=<"noise scaling method imported from zne.scaling.folding">,
+    num_chunks=<"number of chunks to group a large circuit into">,
+)
+```
+
+The hyperparameters that can be controlled are:
+
+
+- `degree`: to modify the extrapolating polynomial
+- `fold_multiplier`: to control how the circuit is scaled
+- `folding_method`: to choose the unitary folding method
+- `num_chunks`: to alter the sampling cost
+
+
+## Controlling how the noise-scaled circuits are created
+
+
+### Extrapolating polynomial
+
+
+Depending on the choice of the degree of the polynomial, how each noise-scaled circuit is scaled varies. This
+in turn influences how the sample matrix for multivariate Richardson extrapolation is created.
+
+
+Suppose we have an input circuit consisting of 4 layers.
```

Comment:
The sample matrix should mostly be hidden from the user, right? I don't think the sentence I suggested to remove below has much value for a reader. If you want to keep it, can we rework it? What do you want them to gain?

```suggestion
The chosen degree of the extrapolating polynomial effects the way in which circuits get scaled, as we'll see below.
For this example, we'll define a circuit consisting of 4 layers.
```